nab/detectors/numenta/numentaNew_detector.py

```diff
@@ -0,0 +1,108 @@
+# ----------------------------------------------------------------------
```

Comment:
I kept that file name because of a method in utils.py that gets called for all detectors in run.py and is supposed to convert filename to classname. I've copy pasted the function below. Let me know what can be a good alternative.

def detectorNameToClass(name):
  name = name[0].upper() + name[1:]
  className = name + "Detector"

  return className